client/me/billing-history/transactions-table.jsx

```diff
@@ -134,7 +134,7 @@ var TransactionsTable = React.createClass( {
 		} else if ( isEmpty( this.state.transactions ) ) {
```

Comment:
I realize that you didn't make the changes here, but maybe you could go ahead and break apart this `if/else`

```js
if ( ! this.state.transations ) {
	return ( … );
}

if ( isEmpty( this.state.transactions ) ) {
	return ( … );
}
```